examples/authentication/server/src/main.rs

```diff
@@ -90,15 +69,14 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let cert = tokio::fs::read(opt.cert).await?;
     let key = tokio::fs::read(opt.private_key).await?;
     let identity = Identity::from_pem(cert, key);
-    let handler = AuthenticationHandler {
-        client_id: opt.client_id.clone(),
-    };
-
     info!("Starting the auth server at {:?}", address);
 
```

Comment:
Perhaps check that client_id and client_secret are non-empty and output an appropriate error message